test/src/unittests/stats/histogram/ranges.txt

```diff
@@ -0,0 +1,3 @@
+ 0.050022757620501929 0.14998849366365799 0.24995422970681405 0.34991996574997009 0.44988570179312615 0.54985143783628221 0.64981717387943827 0.74978290992259433 0.84974864596575039 0.94971438200890645
```

Comment:
Missing the ```input.txt``` file.

Reply:
the path is test/src/unittests/stats/stats/input.txt